format/Schema.fbs

```diff
@@ -204,6 +209,7 @@ union Type {
   Timestamp,
   Interval,
   List,
+  LargeList,
```

Comment:
I think this might be backwards incompatible, because the type is encoded with an enum which is just an integer  (https://google.github.io/flatbuffers/md__schemas.html), you probably need to put this at the end.

Reply:
thanks, fixed!